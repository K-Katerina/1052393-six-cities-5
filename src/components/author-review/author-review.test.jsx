import React from "react";
import renderer from "react-test-renderer";
import AuthorReview from "./author-review";

it(`Should AuthorReview render correctly`, () => {
  const tree = renderer
    .create(
        <AuthorReview
          author={{
            id: 43,
            avatar: `http://picsum.photos/74/74?r=123}`,
            name: `Sun-43`
          }}
        />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
