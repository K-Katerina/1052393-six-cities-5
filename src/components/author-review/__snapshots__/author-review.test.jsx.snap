// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should AuthorReview render correctly 1`] = `
<div
  className="reviews__user user"
>
  <div
    className="reviews__avatar-wrapper user__avatar-wrapper"
  >
    <img
      alt="Reviews avatar"
      className="reviews__avatar user__avatar"
      height="54"
      src="http://picsum.photos/74/74?r=123}"
      width="54"
    />
  </div>
  <span
    className="reviews__user-name"
  >
    Sun-43
  </span>
</div>
`;
